x-pack/plugins/canvas/canvas_plugin_src/functions/common/ply.js

```diff
@@ -103,9 +104,9 @@ export const ply = () => ({
     const datatablePromises = originalDatatables.map(originalDatatable => {
       let expressionResultPromises = [];
 
-      if (args.expression)
+      if (args.expression) {
         expressionResultPromises = args.expression.map(expression => expression(originalDatatable));
-      else expressionResultPromises.push(Promise.resolve(originalDatatable));
+      } else expressionResultPromises.push(Promise.resolve(originalDatatable));
```

Comment:
This violates the styleguide: https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops

I think the requirement of human review for this seems to indicate a problem upstream though.

Reply:
Just for clarity, I'm totally cool with you merging this ahead of an upstream fix as long as you manually fix these.